am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -17,4 +19,12 @@ public void createResourceAccess(String resourceId, String accessorId) {
         jdbi.useExtension(AccessManagementRepository.class,
             dao -> dao.createAccessManagementRecord(resourceId, accessorId));
     }
+
+    public List<String> checkAccess(UserDetails userDetails, String resourceId) {
+        return jdbi.withExtension(AccessManagementRepository.class, dao ->  {
+            List<String> userIds = dao.checkAccess(userDetails.getUserId(), resourceId);
+
+            return (userIds.isEmpty()) ? null : userIds;
```

Comment:
empty list instead of null some the consumer of this method does not need to also check nulls

Reply:
@kpakur 
Please read Scenario 2 https://tools.hmcts.net/jira/browse/AM-88